dbscripts/xml/upgrade.xml

```diff
@@ -210,6 +210,10 @@
 		<note file="docs/release-notes/README-3.4.0" />
 	</upgrade>
 
+	<upgrade minversion="3.0.0.0" maxversion="3.3.9.9">
+		<migration class="lib.pkp.classes.migration.ConvertSiteWideAboutToRichTextArea" />
```

Comment:
This migration can go into the `<upgrade>` element above, just between the `<migration>` and `<note>` elements.